src/player.py

```diff
@@ -1,2 +1,94 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+    def __init__(self, name, curr_room):
+        self.name = name
+        self.curr_room = curr_room
+        self.inventory = []
+
+    def move_n(self):
+        if self.curr_room.n_to == None:
+            print("You hit a wall.")
+            return
+        if self.curr_room.n_to.is_locked == True:
+          # check if user has key
+            print("This room is locked. Unless you have a key.")
+            return
```

Comment:
Good job here. One thing we could do is return the string itself and print out the returned value where the method is called.